en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -392,35 +402,32 @@ plt.ylabel("Number of Agents")
 plt.show()
 ```
 
-{% include figure.html filename="en-or-agent-based-model-communication-networks-03.png" alt="A histogram of the agents and their letters received. This time, with 100 model instantiations running 10 times each, and binned as integers. The y-axis displays the amount of agents that have received a certain amount of letters displayed on the x-axis. In this figure, the distribution is normal, i.e., most agents have an average number of letters received. This indicates that the decision to whom an agent sends a letter is currently random." caption="Figure 3. Histogram of the letters received by all agents after 100 model runs" %}
-
-This runs 100 instantiations of the model, and runs each for 10 steps. (Notice that we set the histogram bins to be integers, since agents can only have whole numbers of letters). By running the model 100 times, we smooth out some of the ‘noise’ of randomness, and get to the model’s overall expected behavior.
+{% include figure.html filename="en-or-agent-based-model-communication-networks-03.png" alt="A histogram of the agents and the number of letters they received. The y-axis displays the number of agents who have received a certain number of letters, displayed on the x-axis." caption="Figure 3. Histogram of the letters received by all agents after 100 model runs" %}
 
-For now, the letter distribution looks close to a normal distribution, or Bell curve, which is expected since the process is random. Let's add some more comparably realistic behavior by introducing space between the agents and let that influence the letter sending decision.
+This runs 100 instantiations of the model, each of which runs for 10 steps. (Notice that we set the histogram [bins](https://en.wikipedia.org/wiki/Data_binning) to integers, since agents can only receive whole numbers of letters.) By running the model 100 times, we smooth out some of the 'noise' of randomness, and approach the model’s overall expected behavior. For now, the letter distribution looks close to a [normal distribution (or bell curve)](https://en.wikipedia.org/wiki/Normal_distribution), which is expected since the process is random.
 
 > _Bonus question 2_:
-Can you rewrite the above function to plot the number of sent letters? What histogram do you expect for that? *Hint*: Are agents sending letters in every round?
+Can you rewrite the function above to plot the number of sent letters? What histogram do you expect for that? *Hint*: Are agents sending letters in every round?
```

Comment:
Could you please clarify what you mean by 'the function above'?

Reply:
We meant the previous code block where the plot is coded - maybe function is misleading here, then. I'd suggest to simply replace it then, so something like:  "Can you rewrite the code block above ..."

Reply:
```suggestion
Can you rewrite the code block above to plot the number of sent letters? What histogram do you expect for that? *Hint*: Are agents sending letters in every round?
```